releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -85,11 +82,22 @@ public void testGenerateOnlyNewFeature() throws Exception {
 
     @Test
     public void testGenerateOnlyBug() throws Exception {
-        final List<String> errors = MainProcess.runPostGenerationAndPublication(
-            createNotes(null, null, createAllNotes(), null, null), createBaseCliOptions("1.0.1")
-                .setGenerateAll(true).build(), true);
+        createAllIssues(BUG);
+        createAllCommits();
 
-        Assert.assertEquals("no errors", 0, errors.size());
+        runMainAndAssertReturnCode(0,
+            "-localRepoPath", getTempFolder().getAbsolutePath(),
+            "-remoteRepoPath", "checkstyle/checkstyle",
+            "-startRef", "12345678",
+            "-releaseNumber", "1.0.1",
+            "-outputLocation", getTempFolder().getAbsolutePath(),
+            "-githubAuthToken", "TOKEN",
+            "-generateAll",
+            "-validateVersion"
+        );
```

Comment:
same big code again and again.
Should be something like:
```
runMainContentGenerationAndAssertReturnCode(0,
            "-releaseNumber", "1.0.1",
            "-generateAll",
            "-validateVersion"
        );
```
to let reader/human to focus on what matters.
if some test need more customization, it can use runMainAndAssertReturnCode.

Reply:
Done.